src/ape_ethereum/transactions.py

```diff
@@ -268,7 +270,9 @@ def decode_logs(
                         decoded_logs.append(library_log)
                 else:
                     decoded_logs.extend(
-                        list(self.provider.network.ecosystem.decode_logs([log], event_abi))
+                        ContractLogContainer(
```

Comment:
Could we override `def extend()` in `ContractLogContainer` to auto-wrap the given iterator so we can do:

```python
decoded_logs.extend(self.provider.network.ecosystem.decode_logs([log], event_abi)
```

Reply:
worked directly